test/SILOptimizer/move_function_kills_copyable_addressonly_lets.swift

```diff
@@ -277,14 +277,3 @@ public func performMoveOnOneEltOfTPair2<T>(_ p: __owned TPair<T>) {
     let _ = _move(p.x) // expected-error {{_move applied to value that the compiler does not support checking}}
     nonConsumingUse(p.y)
 }
-
-/////////
-// Var //
-/////////
-
-public func performMoveOnVar<T>(_ p: T) {
```

Comment:
I deleted this since I am making separate test files for lets/vars since they are different use cases and are handled slightly differently.